project-base/src/Shopsys/ShopBundle/Resources/views/Front/Inline/Newsletter/subscription.html.twig

```diff
@@ -1,4 +1,12 @@
-{{ form_start(form, { attr: { 'data-on-submit': 'ajaxSubmit', 'data-success': success ? 'true' : 'false' }}) }}
+{{ form_start(form, {
+    attr: {
+        'data-on-submit': 'ajaxSubmit',
+        'method': 'GET',
+        'action': '',
+        'data-action': url('front_newsletter_send'),
```

Comment:
I would consider a name that points to the fact that this is fully intentional. Something along the way of `data-ajax-only-action`... I'm just a bit afraid about readability and the possibility that someone will "simplify it" to the previous version in the future.